packages/utils/src/withExport/helpers.js

```diff
@@ -0,0 +1,56 @@
+import get from 'lodash/get';
+import camelCase from 'lodash/camelCase';
+import { downloadFile } from '../helpers';
+
+const CSV_FILE_PREFIX = 'compliance-export';
```

Comment:
Can we change this to `CRC`?
```suggestion
const CSV_FILE_PREFIX = 'CRC';
```